lms/templates/learner_dashboard/course_card_actions.underscore

```diff
@@ -0,0 +1,14 @@
+<% if(enrollment_status){ %>
+    <div class="enrollment-info"><%- gettext('ENROLLED') %></div>
+    <a id="view_course_link" href="<%- course_url %>" class="btn-neutral btn-large view-course-link">
```

Comment:
We should not use ids unless absolutely necessary because of the constraint that they be unique.